test/integration/features/config.feature

```diff
@@ -42,6 +42,7 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
 
     @linux @darwin @windows
     Scenario: CRC config checks (bundle version)
+        Given executing "crc setup" succeeds
```

Comment:
Is this the main reason for this commit? I'm confused by the changes in the previous commit which get almost reverted in this one.

Reply:
Yes, that's the only reason. Without running `crc setup`, bundle won't be unpacked to it's location, and so config setting with that bundle cannot be set. This is not an issue if `basic.feature` is run before this feature (as `crc setup` is executed in basic tests).